features/manage-position/YourPosition.tsx

```diff
@@ -121,37 +121,40 @@ const YourPosition = () => {
           {`${_collateral} ${_collSymbol}`}
         </Status>
         <Status>
-          <Label>Tokens outstanding: </Label>
+          <Label>Tokens created: </Label>
           {`${_tokens} ${_tokenSymbol}`}
         </Status>
         <Status>
           <Label>
-            Estimated identifier price (
+            Identifier price(
             <Link href={pricefeedUrl} target="_blank" rel="noopener noreferrer">
-              Coinbase Pro
+              Coinbase
             </Link>
             ):{" "}
           </Label>
-          {_latestPrice}
+          {Number(_latestPrice).toFixed(4)}
```

Comment:
`_latestPrice` is already passed in as `latestPrice.toFixed(4)`

Reply:
it wasn't though. I added this in to make it scaled correctly. there was perhaps a change in between my PR most recently.

Reply:
not that I can see. looks like this without the toFixed(4)
![image](https://user-images.githubusercontent.com/12886084/88549474-9e349b00-d020-11ea-8bde-13bd2620000f.png)

Reply:
No look at line 89 where this is passed in. It is set to 6 decimals. This was originally set to 6 from 4 because 4 was not enough for ETHBTC

Reply:
Also can you re-add the space back between "price" and "("